README.md

```diff
@@ -48,6 +49,13 @@ A pip package is provided, you can install it with:
 
 There are two optional dependencies GUI and CBR. You can install the optional dependencies by specifying one or more of `GUI`,`CBR` or `all` in braces e.g. `comictagger[CBR,GUI]`
 
+### Chocolatey installation (Windows only)
+
+A [Chocolatey package](https://community.chocolatey.org/packages/comictagger) is provided, you can install it with
+```powershell
+choco install comictagger
+```
```

Comment:
Hi @lordwelch,

Do you mean adding the information like so ?

```suggestion
A [Chocolatey package](https://community.chocolatey.org/packages/comictagger), maintained by @Xav83, is provided, you can install it with:
```powershell
choco install comictagger
```
```

Reply:
Yes